pinot-integration-tests/src/main/java/org/apache/pinot/broker/api/resources/BrokerEchoWithAutoDiscovery.java

```diff
@@ -40,6 +44,7 @@ public class BrokerEchoWithAutoDiscovery {
     public AutoLoadedServiceForTest _injectedService;
     @GET
     @Path("/echo/{table}")
+    @Authorize(targetType = TargetType.TABLE, paramName = "table", action = Actions.Table.ECHO)
```

Comment:
Let's remove this one. This is purely for testing purpose

Reply:
If Authorize annotation is removed, no user may be able call it if default auth returns false. I would advise to change the action to Test or Debug instead.

Reply:
This resource is in the test package for testing purpose only, and is not included in the production code, thus not visible to the user